app/shared-hub/reducers/AntiSuiteReducer.js

```diff
@@ -17,21 +17,27 @@ import {
 	SET_SMART_BLOCK
 } from '../constants/AntiSuiteConstants';
 
-const initialState = {};
+const initialState = {
+	antiSuite: {
+		enable_ad_block: false,
+		enable_anti_tracking: false,
+		enable_smart_block: false
+	}
+};
```

Comment:
I am realizing I have a strong suspicion we don't need the extra `antiSuite` and `setupLifecycle` wrappers at all after all, and that the code would be better and simpler without them....but I can fix that up at the end, don't worry about it unless you want to

Reply:
(Based on our convo, you're right, let's leave it as is)